### PR TITLE
feat: add 'yarn analyze' to inspect the content of the compiled js

### PR DESCRIPTION
**Pull request by riccardo-forina**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "react-virtualized": "^9.20.1"
   },
   "scripts": {
+    "analyze": "find build/static/js -type f -maxdepth 1 -name '*.js' -exec bash -c 'npx source-map-explorer {} &' \\;",
```
[the remark this answers is not in the document]
Comment (gashcrumb) [reply package.json]: I'm off Windows nowadays so it's probably fine for the short term at least.

[the remark this answers is not in the document]
Comment (gashcrumb) [reply package.json]: And actually, I'd just recommend that a Windows dev installes the "subsystem for linux" and use a bash shell for development anyways.

**Comment by gashcrumb** [comment]
BTW, what's the benefit here?  Think that's missing :-)

**Comment by riccardo-forina** [comment]
Right. So it's just a bundle analyzer, it will open a number of tabs (one per each chunk that gets compiled) with an overview of how the file is structured. It's useful to debug the reason why the compiled file is unreasonably big, for example.

**Pull request merged**